nuxeo/nuxeo.py

```diff
@@ -2,16 +2,18 @@
 """ Nuxeo REST API Client. """
 from __future__ import unicode_literals
 
-import base64
 import json
 import logging
 import socket
-import tempfile
-from collections import Sequence
 from urllib import urlencode
+from typing import Any, Dict, List, Optional, Text, Union
 
+import base64
 import requests
+import tempfile
+from collections import Sequence
 from requests import HTTPError
```

Comment:
I was wondering why you did not use `Requests.HTTPError` later in the code instead of importing twice `Requests`?